source/tyr/tyr/command/load_data.py

```diff
@@ -52,9 +52,7 @@ def run(self, instance_name, data_dir):
         data_dirs = data_dir.split(',')
         logging.info("loading data dir : {}".format(data_dirs))
 
-        futur_res = load_data.delay(instance.id, data_dirs)
-
         # this api is only used in artemis and artemis wants to know when this
         # task is finished, so the easiest way for the moment is to wait for
         # the answer
-        futur_res.get()
+        load_data(instance.id, data_dirs)
```

Comment:
If you call the function by `delay`, it will launch a task (aka, send a task to a worker and the worker will do the job in async), it'll cause the case  `launching synchronous subtasks`, by calling it directly, this case is avoided and the function call is blocking which is what we want